packages/portal-proto/src/features/GenomicTables/SomaticMutationsTable/SMTableContainer.tsx

```diff
@@ -145,13 +145,13 @@ export const SMTableContainer: React.FC<SMTableContainerProps> = ({
         geneSymbol: geneSymbol,
         genomicFilters: genomicFilters,
         cohortFilters: cohortFilters,
-        caseFilter: { mode: "", root: {} } as FilterSet,
+        caseFilter: caseFilter,
       });
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [searchTermsForGene, genomicFilters, cohortFilters]);
+  }, [searchTermsForGene, genomicFilters, cohortFilters, caseFilter]);
```

Comment:
I am not sure I understand how `getTopSSM` works in terms of reference equality. Does `useGetSsmTableDataMutation` return the same one every render?

cc. @amlehman @craigrbarnes @alilahrime

Reply:
I agree with Craig. useMutation [docs](https://redux-toolkit.js.org/rtk-query/api/created-api/hooks#usemutation) says the hook subscribes the component to the result and re-renders only as status/data changes. Even if there was multiple getTopSSM references, they should still subscribe to the same result and shouldn't cause unnecessary re-renders by being included in dependency array